tests/integration/test_synapseutils.py

```diff
@@ -345,6 +362,9 @@ def test_copyWiki():
     #Test: entitySubPageId
     third_header = synapseutils.copyWiki(syn, project_entity.id, third_project.id, entitySubPageId=sub_subwiki.id, destinationSubPageId=None, updateLinks=False, updateSynIds=False,entityMap=fileMapping)
     test_ent_subpage = syn.getWiki(third_project.id,third_header[0]['id'])
+    
+    assert_raises(ValueError, synapseutils.copyWiki, syn, project_entity.id, third_project.id, 3)
```

Comment:
I think you accidentally merged your own develop branch (#531) instead of the upstream one.
```
git reset --hard HEAD~
git fetch upstream
git merge upstream/develop
git push -f
```